src/Stats/DateSlider.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+import PropTypes from 'prop-types';
+import { Button } from 'react-native-elements';
+
+export default function SubHeader(props) {
+  const { viewSet, onPressBtn, viewType } = props;
+  const format = viewType === 'month' ? 'MMM YYYY' : 'YYYY';
+
+  return (
+    <View style={{ flexDirection: 'row', justifyContent: 'space-around' }}>
+      <Button
+        containerStyle={{ flex: 1, borderBottomWidth: 2, borderBottomColor: 'white' }}
+        title={viewSet[0].format(format)}
+        type="clear"
+        titleStyle={{ color: 'grey' }}
+        onPress={() => onPressBtn(viewSet[0], viewType)}
+      />
+      <View style={{
+        flex: 1, borderBottomWidth: 2, borderBottomColor: '#30d29d', justifyContent: 'center',
+      }}
+      >
+        <Text style={{ textAlign: 'center', fontSize: 18, fontWeight: '500' }}>{viewSet[1].format(format)}</Text>
+      </View>
+      <Button
+        containerStyle={{ flex: 1, borderBottomWidth: 2, borderBottomColor: 'white' }}
+        title={viewSet[2].format(format)}
+        type="clear"
+        titleStyle={{ color: 'grey' }}
+        onPress={() => onPressBtn(viewSet[2], viewType)}
+      />
+    </View>
+  );
+}
+
+SubHeader.propTypes = {
+  viewSet: PropTypes.instanceOf(Array),
+  onPressBtn: PropTypes.func,
+  viewType: PropTypes.string,
```

Comment:
dont we have some kind of `oneOf`, or `enum`?

Reply:
update it to ` viewType: PropTypes.oneOf(['month', 'year'])`